locales/en-US/server.ftl

```diff
@@ -0,0 +1,214 @@
+homePageSaveShare = Save and Share
+homePageSaveShareDescription = When you take a shot, Firefox posts your screenshot to your online Screenshots library and copies the link to your clipboard. We automatically store your screenshot for two weeks, but you can delete shots at any time or change the expiration date to keep them in your library for longer.
+homePageLegalLink = Legal
+homePagePrivacyLink = Privacy
+homePageTermsLink = Terms
+homePagePrivacyLink = Cookies
+
+// Leave Screenshots page
+
+leavePageConfirmDelete = Confirm account deletion
+leavePageErrorAddonRequired = You must have Firefox Screenshots installed to delete your account
+leavePageErrorGeneric = An error occurred
+leavePageWarning = This will permanently erase all of your Firefox Screenshots data.
+leavePageButtonProceed = Proceed
+leavePageButtonCancel = Cancel
+leavePageDeleted = All of your screenshots have been erased!
+
+// Not Found page
+
+notFoundPageTitle = Page Not Found
+notFoundPageIntro = Oops.
+notFoundPageDescription = Page not found.
+
+// Shot page
+
+shotPageAlertErrorUpdatingExpirationTime = Error saving expiration
+shotPageAlertErrorDeletingShot = Error deleting shot
+shotPageAlertErrorUpdatingTitle = Error saving title
+shotPageConfirmDelete = Are you sure you want to delete this shot permanently?
+shotPageShareButton
+  .title = Share
+shotPageCopy = Copy
+shotPageCopied = Copied
+shotPageShareFacebook
+  .title = Share on Facebook
+shotPageShareTwitter
+  .title = Share on Twitter
+shotPageSharePinterest
+  .title = Share on Pinterest
+shotPageShareEmail
+  .title = Share link via email
+shotPageShareLink = Get a shareable link to this shot:
+shotPagePrivacyMessage = Anyone with the link can view this shot.
+shotPageCopyImageText
+  .label = Copy image text
+shotPageConfirmDeletion = Are you sure you want to delete this shot permanently?
+// Note: {$timediff} is a placeholder for a future relative time clause like 'in 3 days' or 'tomorrow'
+shotPageExpirationMessage = If you do nothing, this shot will be permanently deleted {$timediff}.
+// Note: {$date} is a placeholder for a localized future date as returned by Date.toLocaleString.
+// For example, in en-US, {$date} could be "7/12/2017, 1:52:50 PM".
+shotPageRestoreButton = restore until {$date}
+shotPageExpiredMessage = This shot has expired.
+// Note: This phrase is followed by an empty line, then the URL of the source page
+shotPageExpiredMessageDetails = Here is the page it was originally created from:
+shotPageDeleteButton
+  .title = Delete this shot
+shotPageAbuseButton
+  .title = Report this shot for abuse, spam, or other problems
+shotPageDownloadShot
+  .title = Download
+shotPageDownload = Download
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageScreenshotsDescription = Screenshots made simple. Take, save, and share  screenshots without leaving Firefox.
+shotPageUpsellFirefox = Get Firefox now
+shotPageDoesNotExpire = Does not expire
```

Comment:
It's used [here](https://github.com/mozilla-services/screenshots/pull/3068/files#diff-c089cb65f9a6ab0a43538517ddbcc5b6R525):